### PR TITLE
change titles and add titles for js- files

### PR DESCRIPTION
**Pull request by iheartkode**
Perform SEO on Titles

### CODE REVIEW

File: js-closures.md [hunk 1]
```diff
@@ -10,9 +12,9 @@ function makeFunc() {
...
-```
-
-See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
-
+myFunc();
+```
+
+See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
+
 > tags: closure, javascript, js
```
Comment (Rafase282) [inline js-closures.md]: Did you notice this? Not sure if it was from your previous PR or if it was there before. Would that work?

File: JS-Regex-Resources.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Regular Expressions Resources
+
+
```
Comment (abhisekp) [inline JS-Regex-Resources.md]: 2 blank lines can be removed

File: js-for-loop.md [hunk 3]
```diff
@@ -6,7 +8,7 @@ The javascript `for` statement consists of three expressions and a statement:
 
 * initialization - Run before the first execution on the loop. This expression is commonly used to create counters. Variables created here are scoped to the loop. Once the loop has finished it's execution they are destroyed.
 * condition - Expression that is checked prior to the execution of every iteration. If omitted this expression evaluates to true;
-* final-expression - Expression that is run after every iteration. Usually used to increment a counter.
+* final-expression - Expression that is run after every iteration. Usually used to increment a counter.
```
Comment (abhisekp) [inline js-for-loop.md]: Not sure what change was made in this line.
Most probably, EOL character change. 

> have to check it offline

**Comment by Rafase282** [comment]
@iheartkode remember to restore the `.gitignore`

File: js-for-loop.md (hunk 3 above)
Comment (iheartkode) [inline js-for-loop.md]: yeah maybe line endings, I did not edit that area hmm

File: js-closures.md (hunk 1 above)
Comment (iheartkode) [inline js-closures.md]: I didn't modify it odd

**Comment by Rafase282** [comment]
# LGTM

**Comment by alayek** [comment]
# LGTM :+1:

File: js-closures.md (hunk 1 above)
Comment (Rafase282) [inline js-closures.md]: I don't see any difference ere, he just added a heading, rich different shows just that. Just 2 new lines.

**Pull request merged**